### PR TITLE
update addPeopleSection

### PR DESCRIPTION
**Pull request by DiegoFormoso**
add component People 
update route Main for People Grid

### CODE REVIEW

File: arturito/src/components/PeopleSection/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    {
+        title: 'Birth_year',
+        dataIndex: 'birth_year',
+        key: 'birth_year',
+    },
+    {
+        title: 'Height',
+        dataIndex: 'height',
+        key: 'height',
+        render: (height: string) =>(parseInt( height) / 100) +' centimetros',
```
Comment (soyfriss) [inline arturito/src/components/PeopleSection/index.tsx]: Please, change "centimetros" to "centimeters".

**Review by soyfriss — COMMENTED** [review]
Good job

File: arturito/src/components/PeopleSection/index.tsx (hunk 1 above)
Comment (DiegoFormoso) [reply arturito/src/components/PeopleSection/index.tsx]: Checkuot again bro

**Review by soyfriss — APPROVED** [review]
Excellent

**Pull request merged**